plugins/trace_api_plugin/request_handler.cpp

```diff
@@ -62,6 +62,42 @@ namespace {
 
    }
 
+   fc::variants process_actions(const std::vector<action_trace_v1>& actions, const data_handler_function_v1& data_handler, const yield_function& yield ) {
```

Comment:
A lot of duplicated code, can you make this a template or work off a variant.

Reply:
Combined by template.